.github/workflows/rust-build-and-test.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         else
           BUILD_TAG=$(echo ${{ github.sha }} | head -c 12)
         fi
-        make SHA_TAG="$BUILD_TAG" publish-docker
+        make SHA_TAG="$BUILD_TAG" TAG_LATEST="false" publish-docker
```

Comment:
Leaving this out would work as well, but being explicit might be more readable.

Reply:
I live in fear of defaults